cmd/apiserver-boot/boot/init_repo/repo.go

```diff
@@ -249,17 +249,37 @@ package apis
 `
 
 var workspaceTemplate = `
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_k8s_repo_infra",
+    sha256 = "5ee2a8e306af0aaf2844b5e2c79b5f3f53fc9ce3532233f0615b8d0265902b2a",
+    strip_prefix = "repo-infra-0.0.1-alpha.1",
+    urls = [
+        "https://github.com/kubernetes/repo-infra/archive/v0.0.1-alpha.1.tar.gz",
+    ],
+)
+
+# built-in versions of package rules will no longer be addressed
+# see https://docs.bazel.build/versions/1.1.0/be/pkg.html#deprecated
 http_archive(
-    name = "io_bazel_rules_go",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.6.0/rules_go-0.6.0.tar.gz",
-    sha256 = "ba6feabc94a5d205013e70792accb6cce989169476668fbaf98ea9b342e13b59",
+    name = "rules_pkg",
```

Comment:
`rules_pkg` may be unnecessary since it will be only used by the project which wants to use bazel for packaging `tar.gz`, `.deb` etc.

Reply:
i tried remove it locally but it complains:

> ERROR: /go/src/github.com/yue9944882/apiserver-builder-alpha-test/WORKSPACE:15:1: http_archive rule has no 'name' attribute